[3.1]Revamp overview dropdowns, sliders (#799)
telerik/blazor-docs: components/autocomplete/overview.md

The <a href="https://www.telerik.com/blazor-ui/autocomplete" target="_blank">Blazor AutoComplete component</a> is a textbox that offers the users hints as they type. These suggestions can be [filtered]({%slug autocomplete-filter%}) as the user types. The user can write their own value or click a suggestion from the dropdown to select it and populate the input. You can control the list of suggestions through [data binding]({%slug autocomplete-databind%}), various appearance settings like [dimensions]({%slug common-features/dimensions%}) and [templates]({%slug autocomplete-templates%}).

## Creating AutoComplete

1. Use the `TelerikAutoComplete` tag to add the component to your razor page.

1. Populate the `Data` property with the collection of items that you want to appear in the dropdown.

> 1. [Bind the value of the component]({%slug get-started-value-vs-data-binding %}#value-binding) to a collection of the same type as the collection that you bound through the `Data` property.

Reviewer: ```suggestion
1. [Bind the value of the component]({%slug get-started-value-vs-data-binding %}#value-binding) to the same type as the members in the `Data` parameter.
```

1. (Optional) Enable features like placeholder text and [clear button](#clear-button).

> >caption Creating AutoComplete with two-way value binding, main features, and simple [data binding](data-bind)

Reviewer: "main features" and "simple data binding" don't mean anything
```suggestion
>caption AutoComplete with two-way value binding and [data binding]({%slug autocomplete-databind%}) to a primitive type
```

````CSHTML

> @* Main features and simple data binding for the suggestions and the value *@

Reviewer: ```suggestion
@* AutoComplete with two-way value binding and data binding to a primitive type *@
```

}
````

>tip If you want to get a value identifier for the items in the dropdown instead of their text, consider the [ComboBox component](../combobox/overview). The **AutoComplete** is a **free text** input that accepts any text the user writes, not just the suggestions from the dropdown. Thus, the `Value` of the AutoComplete is always a `string`, while the ComboBox can provide you with a `number` or a `Guid`, not only a `string`.

> ## Component Reference

Reviewer: This section should come later, even last, before Next Steps

The AutoComplete is a generic component and its type is determined by the type of the model you use as its data source. You can find examples in the [Data Bind - Considerations]({%slug autocomplete-databind%}#considerations) article.

## Data Binding

The Blazor AutoComplete @[template](/_contentTemplates/dropdowns/features.md#data-binding) [Read more about the Blazor AutoComplete data binding...]({%slug autocomplete-databind%})

> ## Filter

Reviewer: ```suggestion
## Filtering
```
